src/server/utils/filter.ts

```diff
@@ -0,0 +1,15 @@
+import {Movie} from "../../types"
+
+const sortData = (data: Movie[], type: string, size: string) => {
```

Comment:
I don't see the purpose of `size`, but since it's being used as a number in this function, it would probably be a good idea to type it as a number here, and force any user of the function to make sure that it's a valid number before passing it in.

Right now, this function is responsible for converting the type of the `size` attribute, which is more than it should be responsible for, I think.